framework/model/module/module.go

```diff
@@ -364,9 +396,18 @@ func Main(bundle string, modules []Module) {
 		startup(ctx)
 	}
 
-	log.Infof("starting manager bundle %s with modules %v", bundle, modKinds)
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		log.Errorf("problem running manager, %+v", err)
+	le.AddOnStartedLeading(func(ctx context.Context) {
+		go func() {
+			log.Infof("starting controller manager", bundle, modKinds)
+			if err := mgr.Start(ctx); err != nil {
+				log.Errorf("problem running manager, %+v", err)
+			}
+		}()
+	})
+
+	log.Infof("starting bundle %s with modules %v", bundle, modKinds)
+	if err := le.Run(ctrl.SetupSignalHandler()); err != nil {
+		log.Errorf("problem running, %+v", err)
 		stop <- struct{}{}
```

Comment:
seems to not been used anywhere else. meaningless?

Reply:
yes, fix this